recipes/expected-lite/all/conanfile.py

```diff
@@ -18,7 +18,7 @@ def _source_subfolder(self):
         return "source_subfolder"
 
     def configure(self):
-        if self.settings.compiler.cppstd:
+        if tools.valid_min_cppstd(self, 11):
```

Comment:
```suggestion
        if self.settings.compiler.cppstd:
```

Previous logic should be restored. Here it's broken. If `tools.valid_min_cppstd(self, 11)` is True, it doesn't make sense to call `tools.check_min_cppstd(self, 11)` afterwards.

Reply:
@SpaceIm Thanks for your feedback.

Wondering what would be the proper way to prevent the following error with Windows/Visual Studio:

```
prompt>conan create . expected-lite/0.5.0@
...
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=Visual Studio
compiler.runtime=MD
compiler.version=15
os=Windows
os_build=Windows
[options]
[build_requires]
[env]

ERROR: expected-lite/0.5.0: Error in configure() method, line 21
        if self.settings.compiler.cppstd:
        ConanException: 'settings.compiler.cppstd' doesn't exist for 'Visual Studio'
'settings.compiler' possible configurations are ['runtime', 'toolset', 'version']
```

Reply:
It shouldn't happen with a non-corrupted `settings.yml` (or recent conan version).

Reply:
Thanks!